frontend/src/lib/constants.tsx

```diff
@@ -132,6 +132,8 @@ export const FEATURE_FLAGS = {
     VARIANT_OVERRIDES: 'variant-overrides', // owner: @neilkakkar
     ONBOARDING_V2_EXPERIMENT: 'onboarding-v2-experiment', // owner: #team-growth
     RECORDING_AUTOPLAY: 'recording-autoplay', // owner: #team-session-recordings
+    SIGNUP_FORM_EXPERIMENT: 'signup-form-experiment', // owner: #team-growth
```

Comment:
why is this here

Reply:
Because I merged incorrectly, it seems like